app/templates/start.html

```diff
@@ -5,4 +5,19 @@ <h1>START</h1>
 <p>welcome! this is the start page... sign in</p>
 <p>we can have the logo and i guess our project can sit here until user is ready to log in</p>
 <a href="{{ url_for('login') }}">Go to Login</a>
+<p>
+
+        <a href="{{ url_for('login') }}">
+        <img src="{{url_for('static', filename='acct.jpg')}}" align="middle" style="width:50px;height:50px;"></a>
```

Comment:
nit: avoid inline styles

Reply:
gotcha, I think ill comment this out since we're using the nav bar instead

Reply:
correction: I added it to a txt file, in case we want to use it later